noxfile.py

```diff
@@ -0,0 +1,73 @@
+"""Nox config."""
+
+from __future__ import annotations
+
+import nox
+
+# Options to modify nox behaviour
+nox.options.default_venv_backend = "uv|virtualenv"
+nox.options.reuse_existing_virtualenvs = True
+nox.options.sessions = ["lint", "tests"]
+
+ALL_PYTHON = ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
Seems silly but then only one line change when we drop support for one
```suggestion
ALL_PYTHON = [
	"3.8",
	"3.9",
	"3.10",
	"3.11",
	"3.12"
]
```